rust/datafusion/src/logical_plan/dfschema.rs

```diff
@@ -89,11 +89,11 @@ impl DFSchema {
     }
 
     /// Create a `DFSchema` from an Arrow schema
-    pub fn from(schema: &Schema) -> Result<Self> {
+    pub fn try_from(schema: Schema) -> Result<Self> {
```

Comment:
renamed this `try_from` to be more consistent with Rust conventions of conversions that can fail

Reply:
There is a trait for it: https://doc.rust-lang.org/std/convert/trait.TryFrom.html

Reply:
That is an excellent point -- I will update the code to use that.

Reply:
https://github.com/apache/arrow/pull/8888/commits/18fb41314069df87701faf31a6685202a80fa8a1